sale_product_multi_add/__manifest__.py

```diff
@@ -9,7 +9,7 @@
     "category": "Sale Management",
     "version": "15.0.1.0.0",
     "license": "AGPL-3",
-    "depends": ["sale"],
+    "depends": ["sale_stock"],
```

Comment:
@st-anagpal Could you explain why you need this dependency?

Reply:
> @rousseldenis Yes, a field 'picking_policy' is used in test cases which is defined in the module sale_stock. The test case was failing for this module earlier and in order to fix it I added picking_policy and this fixed the issue.

Test failing is not a good reason to introduce a new dependency. You should be able to install sale_product_mutli_add module without stock one!

Reply:
@rousseldenis Ok noted. I'll remove this dependency but in that case the test cases would fail. What is the best way out here?

Reply:
@n17 What are your thoughts here? Any suggestions?

Reply:
> @rousseldenis Ok noted. I'll remove this dependency but in that case the test cases would fail. What is the best way out here?

If you speak about your tests code (as here there is no error), that depends on how you build them. Maybe you pull a dependency there that requires the picking policy. So, you need to adapt your tests

Reply:
@rousseldenis Ok thank you. It is done.